### PR TITLE
Include OpenZiti as a zero trust overlay network

### PR DESCRIPTION
**Pull request by dovholuknf**
Protect your services using a zero trust overlay network. Excellent for self-hosted virtual private servers (VPS). Generic enough to support any service/project you want to expose to friends, family, colleagues, etc

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [ ] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by fbartels — CHANGES_REQUESTED** [review]
>  Any category you are creating has the minimum requirement of 3 items.
If not, your addition may be inserted into Misc/Other.

This requirement is not fulfilled by this PR.

**Comment by dovholuknf** [comment]
Oh my misunderstanding. That makes perfect sense now, for some reason I thought that referred to the github labels. But this makes way more sense. Is it acceptable to move other rows from misc. to here or is it more appropriate to just list two others?

I moved it under misc. for now thank you!

**Comment by dovholuknf** [comment]
🤦 fixing

**Comment by dovholuknf** [comment]
thank you. fixed.

**Review by fbartels — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1248,6 +1248,7 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [Noisedash](https://github.com/kaythomas0/noisedash) - Self-hostable web tool for generating ambient noises/sounds using audio tools and user-uploadable samples. `AGPL-3.0` `Nodejs`
 - [Notica](https://notica.us) - Lets you send browser notifications from your terminal to your desktop or phone. No installation or registration is required. ([Source Code](https://github.com/tannercollin/Notica)) `MIT` `Nodejs`
 - [Ombi](https://ombi.io/) - A content request system for Plex/Emby, connects to SickRage, CouchPotato, Sonarr, with a growing feature set. ([Demo](https://app.ombi.io/), [Source Code](https://github.com/Ombi-app/Ombi)) `GPL-2.0` `C#`
+- [OpenZiti](https://openziti.github.io/) - Fully-featured, self-hostable, zero trust overlay network. Includes a full mesh overlay network, includes 2FA support out of the box, includes clients for all major desktop/mobile OS'es. Build zero trust network principles into your application via SDKs. ([Source Code](https://github.com/openziti/ziti)) `Apache-2.0` `Go/C/Java/Android/Swift/Node/C#`
```
[the remark this answers is not in the document]
Comment (dovholuknf) [reply README.md:1251]: Go is what the 'server side' is written in so I can update that. The other's are the "client side" languages available. I'll update to `go`. Thanks

**Comment by dovholuknf** [comment]
@nodiscc - changed implemented. thank you!

**Comment by dovholuknf** [comment]
Howdy @nodiscc , I made the requested change. Hopefully, I've gotten it all sorted now

**Review by nodiscc — APPROVED** [review]

**Comment by dovholuknf** [comment]
Thank you, @nodiscc, for approving. What's the process for merging this back now? I expect that it's just whenever someone come around to check for mergable prs?

**Pull request merged**